src/plugins/query_enhancements/server/search/ppl_async_search_strategy.ts

```diff
@@ -58,7 +59,10 @@ export const pplAsyncSearchStrategyProvider = (
           } as IDataFrameResponse;
         } else {
           request.params = { queryId: inProgressQueryId };
-          const queryStatusResponse: any = await pplAsyncJobsFacet.describeQuery(context, request);
+          const queryStatusResponse: FacetResponse = await pplAsyncJobsFacet.describeQuery(
+            context,
+            request
+          );
 
           if (!queryStatusResponse.success) handleFacetError(queryStatusResponse);
 
```

Comment:
the line below this has `queryStatusResponse?.data`, based on type the `?` should be unnecessary

Reply:
updated! [306cd4f](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8781/commits/306cd4f42a1d5db19eed7f2860890c57f34de49a)